core/templates/installation.php

```diff
@@ -1,11 +1,13 @@
 <?php
 script('core', 'install');
+
+$urlGenerator = $_['urlGenerator'];
 ?>
 <input type='hidden' id='hasMySQL' value='<?php p($_['hasMySQL']) ?>'>
 <input type='hidden' id='hasSQLite' value='<?php p($_['hasSQLite']) ?>'>
 <input type='hidden' id='hasPostgreSQL' value='<?php p($_['hasPostgreSQL']) ?>'>
 <input type='hidden' id='hasOracle' value='<?php p($_['hasOracle']) ?>'>
-<form action="index.php" method="post" class="guest-box install-form">
+<form action="<?php $urlGenerator->getAbsoluteURL("index.php") ?>" method="post" class="guest-box install-form">
```

Comment:
```suggestion
<form method="post" class="guest-box install-form">
```

Would be my favorite.

Reply:
I wasn't sure on this since the template isn't directly linked to `index.php`. But giving it a second thought setting it to `index.php` makes assumptions about the location of the form as well so yeah that's probably the best solution.